model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/group/entity/JpaGroupEntity.java

```diff
@@ -62,7 +61,8 @@ public class JpaGroupEntity extends AbstractGroupEntity implements JpaRootVersio
     @Column
     private int version;
 
-    @Type(value = UserTypeLegacyBridge.class, parameters = @Parameter(name = UserTypeLegacyBridge.TYPE_NAME_PARAM_KEY, value = "jsonb"))
+//    @Type(value = UserTypeLegacyBridge.class, parameters = @Parameter(name = UserTypeLegacyBridge.TYPE_NAME_PARAM_KEY, value = "jsonb"))
+    @Type(value = JsonbType.class)
```

Comment:
Here's a commented out line that could be removed. 

The annotation can be written shorter without the `value=` as `@Type(JsonbType.class)` as the value is the default parameter.